src/System.ComponentModel.Annotations/tests/CustomValidationAttributeTests.cs

```diff
@@ -77,7 +77,17 @@ public static void Constructor(Type validatorType, string method)
         public static void RequiresValidationContext_Get_ReturnsExpected(string method, bool expected)
         {
             CustomValidationAttribute attribute = GetAttribute(method);
-            Assert.Equal(expected, attribute.RequiresValidationContext);
+
+            // The full .NET framework has a bug where CustomValidationAttribute doesn't
+            // validate the context. See https://github.com/dotnet/corefx/pull/5203.
```

Comment:
Please file an issue describing this difference in behavior. (#3794 and #5203 don't seem to have any associated issue.)